Pod/Tests/TestStash.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+import XCTest
```

Comment:
Although I'm not the guy to ask for tests, don't know how to write them... :cry: